src/generator/AutoRest.CSharp.Unit.Tests/Bug1161.cs

```diff
@@ -53,10 +54,16 @@ public Bug1161(ITestOutputHelper output) : base(output)
                 // Should also succeed.
                 Assert.True(result.Succeeded);
 
-                var recurrenceSchedule = fileSystem.ReadFileAsText(@"GeneratedCode\Models\RecurrenceSchedule.cs");
-                Assert.NotNull(recurrenceSchedule);
-
+                // try to load the assembly
+                var asm = Assembly.Load(result.Output.GetBuffer());
+                Assert.NotNull(asm);
                 
+                // verify that we have the class we expected
+                var recurrenceScheduleObject = asm.ExportedTypes.FirstOrDefault(each => each.FullName == "Test.Models.RecurrenceSchedule");
+                Assert.NotNull(recurrenceScheduleObject);
+                // verify the property is generated 
+                var prop = recurrenceScheduleObject.GetProperty("WeekDays");
+                Assert.NotNull(prop);
```

Comment:
check the type (ie, prop.GetType() and make sure it is the expected type, and that it is nullable)